src/PyCall.jl

```diff
@@ -334,14 +334,63 @@ end
 
 #########################################################################
 
+if is_windows()
+    # Many python extensions are linked against a very specific version of the
+    # MSVC runtime library. To load this library, libpython declares an
+    # appropriate manifest, but unfortunately most extensions do not.
+    # Libpython itself does extend its activation context to any extensions it
+    # loads, but some python libraries (e.g. libzmq), load such extensions
+    # through ctypes, which does not have this functionality. Work around
+    # this by manually activating the activation context before any call to
+    # PyImport_ImportModule, since extensions are most likely to be loaded
+    # during import.
+
+    immutable ACTIVATION_CONTEXT_BASIC_INFORMATION
+        handle::Ptr{Void}
+        dwFlags::UInt32
+    end
+    const ActivationContextBasicInformation = 1
```

Comment:
The [QueryActCtxW docs](https://msdn.microsoft.com/en-us/library/aa375704(v=vs.85).aspx) list this constant as "not available" ... ?